packages/devtools/lib/src/framework/framework.dart

```diff
@@ -341,8 +344,8 @@ abstract class Screen {
   final String id;
   final String iconClass;
   final StatusItem helpStatus;
-  final bool disabled;
   final String disabledTooltip;
+  bool disabled;
```

Comment:
I would fix the logic where you are setting the Screen as disabled in the first place to use the helper you added to determine that the screen should never been disabled.

Reply:
added logic to constructor and made disabled final again.